subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/DefaultJvmPluginServices.java

```diff
@@ -117,6 +106,22 @@ public <T> void configureAsRuntimeClasspath(HasConfigurableAttributes<T> configu
         );
     }
 
+    @Override
+    public void configureAsApiElements(Configuration configuration) {
```

Comment:
Why does this have a different signature than `configureAsCompileClasspath()` and `configureAsRuntimeClasspath()`?  (i.e. `Configuration` vs `HasConfigurableAttributes`)

Reply:
A previous version of the method required a full `Configuration`, but it has been since simplified. I've updated it to use a `HasConfigurableAttributes`